spec/project_spec.rb

```diff
@@ -166,7 +166,7 @@ def count_processes(text)
         # Timeout doesn't kill the running process
         # i.e. see fastlane/fastlane_core#102
         expect(count_processes(text)).to eq(count + 1)
-        sleep(3)
```

Comment:
This looks ok, another change that would make this more bullet proof would be to make `text` be `SecureRandom.uuid[0..7]` or something similar to prevent the case that an old process is somehow getting counted.